kaizen/cli/commands/utils/confirmation_manager.py

```
@@ -0,0 +1,296 @@
+            
+        Returns:
+            True if PR creation should proceed, False otherwise
+        """
+        self.console.print("\n[bold yellow]⚠ Pull Request Creation Confirmation[/bold yellow]")
+        self.console.print("Auto-fix has been completed. A pull request will now be created with the fixes.")
+        self.console.print(f"Base branch: {base_branch}")
+        self.console.print(f"PR strategy: {pr_strategy}")
+        
+        # Show auto-fix summary
+        if test_result.test_attempts:
+            self.console.print(f"\n[bold]Auto-fix summary:[/bold]")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)